pylintrc

```diff
@@ -8,4 +8,4 @@ class-rgx=[A-Z_][_a-zA-Z0-9]+$
 ignored-modules = responses,requests
 
 [MESSAGES CONTROL]
-disable=missing-docstring,too-many-instance-attributes,attribute-defined-outside-init,too-few-public-methods,dangerous-default-value,duplicate-code,bad-continuation,useless-object-inheritance,too-many-public-methods
+disable=missing-docstring,too-many-instance-attributes,attribute-defined-outside-init,too-few-public-methods,dangerous-default-value,duplicate-code,bad-continuation,useless-object-inheritance,too-many-public-methods,R1735,C0209,W0707
```

Comment:
I think we should use `use-dict-literal`, `consider-using-f-string` and `raise-missing-from` instead of the codes that are less descriptive.

Reply:
Yes but lets do this in the follow up PR, because this will introduce lots of changes across codebase.